includes/Core/Util/Plugin_Status.php

```diff
@@ -11,80 +11,56 @@
 namespace Google\Site_Kit\Core\Util;
 
 /**
- * Plugin_Status class.
+ * Utility class for checking the status of plugins.
  *
  * @since n.e.x.t
+ * @access private
+ * @ignore
  */
 class Plugin_Status {
 
 	/**
-	 * Plugin installed identifier.
+	 * Determines whether a plugin is active.
 	 *
 	 * @since n.e.x.t
-	 */
-	const PLUGIN_STATUS_INSTALLED = 'installed';
-
-	/**
-	 * Plugin not active identifier.
 	 *
-	 * @since n.e.x.t
+	 * @param string $plugin Path to the plugin file relative to the plugins directory.
+	 * @return bool
 	 */
-	const PLUGIN_STATUS_ACTIVE = 'active';
+	public static function is_plugin_active( $plugin ) {
```

Comment:
This obviously "conflicts" with the runtime checks we already have but is useful as a utility wrapper by allowing it to be called safely.